airly/_private.py

```diff
@@ -27,11 +27,18 @@ def __init__(self, api_key, session: aiohttp.ClientSession, base_url,
             self.headers['Accept-Language'] = language
         self.base_url = base_url
         self.session = session
+        self.requests_per_day = None
+        self.requests_remaining = None
 
     async def get(self, request_path):
         url = self.base_url + request_path
         _LOGGER.debug("Sending request: " + url)
         async with self.session.get(url, headers=self.headers) as response:
+            if response.status in (200, 429):
```

Comment:
this `if` is unnecessary as you check whether headers are present anyway.

Reply:
Yeah, I thought the same... but few hours after committing the change.